### DISCUSSION TITLE
无法登录专业版

### DISCUSSION THREAD

**Discussion by waterstars19** [getlantern/lantern-forum-cn#391] [General]
3月4日付费的，买了两年的，但是邮箱收不到验证码，其他设备也都登陆不了！

账号：waterstars19@hotmail.com

**Comment by MaxMKMax** [comment]
@waterstars19 
好的，您也可以给我您的付款收据吗？我将帮助您与团队联系并要求他们手动向您发送代码。请几天后查看您的电子邮件或 github。

给您带来的不便敬请谅解，感谢您对蓝灯的支持!

**Reply by waterstars19** [reply]
![Screenshot_20240407_181727_com eg android AlipayG](https://github.com/getlantern/lantern-forum-cn/assets/112752973/dfae7f98-c20f-4b25-9aa5-351b3ab54edb)
@Mike777Khoo 这是付款的信息

**Reply by XINYUAN1986** [reply]
我的这两天也出问题了，邮箱可以收到验证码，但是无法登陆专业版。输入验证码后显示授权成功，但是跳转到主页后依然后试用版。
